site/content/docs/include/common-svc-fields.md

```diff
@@ -158,15 +158,30 @@ Required. Specify the location in the container where you would like your volume
 <span class="parent-field">volume.</span><a id="read_only" href="#read-only" class="field">`read_only`</a> <span class="type">Bool</span>  
 Optional. Defaults to `true`. Defines whether the volume is read-only or not. If false, the container is granted `elasticfilesystem:ClientWrite` permissions to the filesystem and the volume is writable.
 
-<span class="parent-field">volume.</span><a id="efs" href="#efs" class="field">`efs`</a> <span class="type">Map</span>  
-Specify more detailed EFS configuration.
+<span class="parent-field">volume.</span><a id="efs" href="#efs" class="field">`efs`</a> <span class="type">Bool or Map</span>  
```

Comment:
nit: for consistency, can we use the full "Boolean" here and elsewhere in the `volumes` fields?